### ISSUE TITLE
BAZARR unable to index external subtitles.

### ISSUE THREAD

**Issue by JpF4E** [morpheus65535/bazarr#1170]
This appears when I run the "Search for wanted Movies Subtitles" task.

```
Message
BAZARR unable to index external subtitles.
Exception
Traceback (most recent call last):
 File "/opt/bazarr/bazarr/list_subtitles.py", line 162, in store_subtitles_movie
  subtitles = guess_external_subtitles(full_dest_folder_path, subtitles)
 File "/opt/bazarr/bazarr/list_subtitles.py", line 471, in guess_external_subtitles
  if language.forced:
AttributeError: \NoneType\ object has no attribute \forced\
```
Using v0.9.0.5.

**Comment by morpheus65535** [comment]
Fixed in v0.9.0.6 in dev branch for now.

**Comment by JpF4E** [comment]
It appears to have been fixed.
Thanks.
